doc/source/whatsnew/v0.18.1.txt

```diff
@@ -43,8 +43,8 @@ Enhancements
 
 API changes
 ~~~~~~~~~~~
-
-
+- ``searchsorted`` for ``Index`` and ``TimedeltaIndex`` now accept a ``sorter`` argument to maintain compatibility with numpy's ``searchsorted`` function (:issue:`12238`)
+- The parameter for specifying values to be inserted into a ``pandas`` object when calling its ``searchsorted`` method is now uniformly denoted as ``value``
```

Comment:
pls just make the whatsnew simpler. just say the signature changed the names of the arguments and put in a pointer to the signature.